src/axom/inlet/Field.cpp

```diff
@@ -64,6 +64,126 @@ bool Field::required()
   return (bool)intValue;
 }
 
+std::shared_ptr<Field> Field::addDefaultString(std::string value) {
+  SLIC_ASSERT_MSG(m_type == FieldType::STRING, "Default value type did not match STRING");
+
+  if (m_sidreGroup->hasView("defaultValue"))
+  {
+    std::string msg = fmt::format("Inlet Field has already defined default value: {0}",
+                                  m_sidreGroup->getName());
+    SLIC_WARNING(msg);
```

Comment:
Just a thought.  Maybe we should be setting a global error flag at the base of the given root sidre group level anytime there is a warning thrown at the in defining the schema.  It could then be checked in the verify function.